client/blocks/upload-image/constants.js

```diff
@@ -1 +1,25 @@
-export const ALLOWED_FILE_EXTENSIONS = [ 'jpeg', 'gif', 'png', 'bmp', 'tiff', 'ico' ];
+/**
+ * External dependencies
+ */
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { ValidationErrors } from 'lib/media/constants';
+
+export const ALLOWED_FILE_EXTENSIONS = [ 'jpg', 'jpeg', 'gif', 'png', 'bmp', 'tiff', 'ico' ];
+
+export const ERROR_UNSUPPORTED_FILE = 'ERROR_UNSUPPORTED_FILE';
+export const ERROR_IMAGE_EDITOR_DONE = 'ERROR_IMAGE_EDITOR_DONE';
+export const ERROR_UPLOADING_IMAGE = 'ERROR_UPLOADING_IMAGE';
+
+export const ERROR_STRINGS = {
```

Comment:
Fair point. We should keep the error codes in the `constants` file since they can be imported elsewhere (and actions performed based on which error code was returned).

But for the error strings: yeah, they are probably not reusable outside of `upload-image`.

I'll update accordingly.